app/views/static_pages/help.html.erb

```diff
@@ -8,7 +8,7 @@
 <div class="row">
   <div class="col-md-12">
     <div>
-        <p><%= _("When you log in to DMPRoadmap you will be directed to the 'My Dashboard' page. From here you can edit, share, download, copy or remove any of your plans. You will also see plans that have been shared with you by others.") %></p>
+        <p><%= _("When you log in to %{application_name} you will be directed to the 'My Dashboard' page. From here you can edit, share, download, copy or remove any of your plans. You will also see plans that have been shared with you by others.") % { :application_name => Rails.configuration.x.application.name } %></p>
```

Comment:
Prefer the use of `ApplicationService.application_name` here over access the config value directly.

Reply:
Change made. I was trying to be consistent. You will need to let me know why.